src/mame/video/fmtowns.cpp

```diff
@@ -267,10 +269,13 @@ READ8_MEMBER( towns_state::towns_video_cff80_r )
 			else
 				return 0x00;
 		case 0x06:
+			ret |= 0x10;
+			xpos = machine().first_screen()->hpos();
```

Comment:
Any particular reason you're using machine().first_screen() instead of m_screen?

Reply:
No particular reason, as I said the code that checks for the position is basically the same as the existing one for a different I/O port, so I did it the same way for consistency. The driver only supports one screen anyway, so if m_screen is preferable I have no problem with changing it.

Reply:
AFAIK it is, though someone please correct me if I'm wrong.